### PR TITLE
Refuse to let things be broken, hard or complex.

### PR DESCRIPTION
**Pull request by johnboxall**
From this Slack convo: https://mobify.slack.com/archives/engineering/p1453323999003300

With more and more new Mobifiers, I worry that coming into the company, people will have the impression that things work a certain way for a good reason, and that they can't get any better or can't be changed.

That just isn't the case.

Most things we do because someone implemented snap decision (experimentation over argument) and it worked well enough that we rolled it out.

On the path from good to great, we need to question everything. Most of the systems that we built as small startup aren't going to scale to the next level of growth.

I want everyone to question how we do things and ask if something can be improved for the modern Mobify.

I've tried to capture this in a way such that it could be widely applicable outside of engineering.

The one thing that feels a bit off here, is that I think the best Mobifiers are solution oriented. They leave the campground better than they found it. I'm not sure this says it yet.

### CODE REVIEW

**Comment by jansepar** [comment]
Is there anywhere that could fit within the "Mobify Values" section? It would be great if we could expand on that with explanations and examples!

**Comment by johnboxall** [comment]
@jansepar took another swing at it– let me know what you think!

File: README.md [hunk 1]
```diff
@@ -231,11 +241,19 @@ repeating the mistakes of the past.
 - Extract common or shared functionality into utility functions, and if needed across multiple projects, extract into libraries or frameworks.
 - Re-use the things you've built and avoid copying and pasting. For example we don't want duplicated functionality in our functions, classes, modules, systems. We also want to avoid duplication and share code across sites/apps/systems.
 
-#### The Boy Scouts Rule
-- Leave the campground better than you've found it.  Always check a module in cleaner than you found it.
-- Try to improve code quality every time you work on a project.
-- Fix formatting, spelling, do small refactorings…  But do this in moderation - if it's too hard to actually see the meat of the changes, you should probably move your cleanup changes to a separate branch.
-- Continuous improvement through iteration: Improve the code base slowly but surely.
+#### Scouts Rule
```
Comment (jansepar) [inline README.md]: Huge +1 making this gender-neutral :)

**Comment by jansepar** [comment]
Looks pretty good! :+1: 

Would be curious to hear why stripped it from Zen completely.

**Comment by johnboxall** [comment]
@jansepar I think the Zen should be static-ish. If we change it often, it loses its effect– so I'd like to reserve changes to it to things we feel are really *really* **really** important changes to be aware of. 

I think the point already had adequate coverage – so lets see how things go, and we can always revisit putting it in.

**Pull request merged**